tests/acceptance/features/bootstrap/PublicWebDavContext.php

```diff
@@ -1207,11 +1207,7 @@ public function tryingToDownloadUsingWebDAVAPI(
 
 		$responseContent = $response->getBody()->getContents();
 		\libxml_use_internal_errors(true);
-		Assert::assertNotFalse(
-			\simplexml_load_string($responseContent),
-			"response body is not valid XML, maybe download did work\n" .
-			"response body: \n$responseContent\n"
-		);
+		Assert::assertEmpty($responseContent, "response body is not empty, maybe download did work");
```

Comment:
maybe the response is xml body for failure cases. So `Assert::assertNotFalse(\simplexml_load_string(` was required

Reply:
yes, when we have 401 -> response is xml
<img width="759" alt="image" src="https://github.com/owncloud/ocis/assets/84779829/01799380-8f29-46e0-b28c-2bb69369ff3a">

but if we have 403 (secret file drop) or 404 (file is deleted or not exist) -> we have empty response (no xml)
<img width="786" alt="image" src="https://github.com/owncloud/ocis/assets/84779829/6bd812df-20a3-4a12-a517-d8b6348ea96b">

Reply:
I reverted my changes. It should work now. I don't check that response body is empty (I think it's not needed)